src/main/resources/com/rackspace/jenkins_nodepool/NodePoolSlave/configure-entries.jelly

```diff
@@ -0,0 +1,41 @@
+<!--
+  Config page
+-->
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout"
+         xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+
+    NodePool agents/slaves are ephemeral and last for the duration of a single build.
+    <p/>
+
+    <f:entry title="${%Hold Node?}" field="held">
+        <f:checkbox/>
+    </f:entry>
+
+    <f:section title="Connectivity">
+
+        <f:block>
+            <p><b>IP address:</b> ${it.nodePoolNode.host}</p>
+            <p><b>Port:</b> ${it.nodePoolNode.port}</p>
+        </f:block>
+    </f:section>
+
+    <f:section title="Labels">
+
+        <f:block>
+            <p><b>Jenkins label:</b> ${it.nodePoolNode.jenkinsLabel}</p>
+            <p><b>NodePool label:</b> ${it.nodePoolNode.nPType}</p>
+        </f:block>
+
+    </f:section>
+
+    <f:section title="ZooKeeper">
+        <f:block>
+            <p><b>NodePool connection string: </b> ${it.nodePoolNode.nodePool.connectionString}</p>
+            <p><b>ZooKeeper Path:</b> /nodepool${it.nodePoolNode.path}</p>
+            <p><b>ZooKeeper ID:</b> ${it.nodePoolNode.zKID}</p>
```

Comment:
❤️ all the useful info

Reply:
That was the goal, yeah.

I debated using NodeProperty fields instead, and that might still be the right way to go, but they are read-write fields..and we don't need edits on these.